pages/index.jsx

```diff
@@ -31,6 +32,8 @@ export default () => {
   });
 
   useEffect(() => {
+    document.body.setAttribute("dir", "rtl");
```

Comment:
Use _document file for setting direction

[Check This out](https://nextjs.org/docs/advanced-features/custom-document)